### ISSUE TITLE
Issue installing frida on windows

### ISSUE THREAD

**Issue by P3pito** [frida/frida-website#50]
Hi,

I'm trying to install frida. 
I followed instructions and run the "pip install frida" in sudo mode (run 'cmd' as admin) and got this error message: 

> "There is no item named '_frida.pyd' in the archive"

Here is the full log:

```
C:\WINDOWS\system32>pip install frida
Collecting frida
  Using cached frida-7.1.6.zip
Requirement already satisfied (use --upgrade to upgrade): colorama>=0.2.7 in c:\program files\python34\lib\site-packages (from frida)
Requirement already satisfied (use --upgrade to upgrade): prompt-toolkit>=0.57 in c:\program files\python34\lib\site-packages (from frida)
Requirement already satisfied (use --upgrade to upgrade): pygments>=2.0.2 in c:\program files\python34\lib\site-packages (from frida)
Requirement already satisfied (use --upgrade to upgrade): wcwidth in c:\program files\python34\lib\site-packages (from prompt-toolkit>=0.57->frida)
Requirement already satisfied (use --upgrade to upgrade): six>=1.9.0 in c:\program files\python34\lib\site-packages (from prompt-toolkit>=0.57->frida)
Installing collected packages: frida
  Running setup.py install for frida ... error
    Complete output from command "c:\program files\python34\python.exe" -u -c "import setuptools, tokenize;__file__='C:\\Users\\Kermit\\AppData\\Local\\Temp\\pip-build-kopnyw58\\frida\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\Kermit\AppData\Local\Temp\pip-7l3e3k_6-record\install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.4
    creating build\lib.win-amd64-3.4\frida
    copying frida\application.py -> build\lib.win-amd64-3.4\frida
    copying frida\core.py -> build\lib.win-amd64-3.4\frida
    copying frida\discoverer.py -> build\lib.win-amd64-3.4\frida
    copying frida\lsd.py -> build\lib.win-amd64-3.4\frida
    copying frida\ps.py -> build\lib.win-amd64-3.4\frida
    copying frida\repl.py -> build\lib.win-amd64-3.4\frida
    copying frida\tracer.py -> build\lib.win-amd64-3.4\frida
    copying frida\__init__.py -> build\lib.win-amd64-3.4\frida
    running build_ext
    querying pypi for available prebuilds
    downloading prebuilt extension from https://pypi.python.org/packages/00/8c/3b3d3887b5b530b696c29d812b367743957034e0d36299b49dad3bf405ac/frida-7.1.6-py3.5-win-amd64.egg
    extracting prebuilt extension
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Kermit\AppData\Local\Temp\pip-build-kopnyw58\frida\setup.py", line 157, in <module>
        'build_ext': FridaPrebuiltExt
      File "c:\program files\python34\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "c:\program files\python34\lib\distutils\dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "c:\program files\python34\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\program files\python34\lib\site-packages\setuptools\command\install.py", line 61, in run
        return orig.install.run(self)
      File "c:\program files\python34\lib\distutils\command\install.py", line 539, in run
        self.run_command('build')
      File "c:\program files\python34\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "c:\program files\python34\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\program files\python34\lib\distutils\command\build.py", line 126, in run
        self.run_command(cmd_name)
      File "c:\program files\python34\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "c:\program files\python34\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\program files\python34\lib\site-packages\setuptools\command\build_ext.py", line 49, in run
        _build_ext.run(self)
      File "c:\program files\python34\lib\distutils\command\build_ext.py", line 339, in run
        self.build_extensions()
      File "c:\program files\python34\lib\distutils\command\build_ext.py", line 448, in build_extensions
        self.build_extension(ext)
      File "C:\Users\Kermit\AppData\Local\Temp\pip-build-kopnyw58\frida\setup.py", line 106, in build_extension
        egg_zip.extract(os.path.basename(target), target_dir)
      File "c:\program files\python34\lib\zipfile.py", line 1223, in extract
        member = self.getinfo(member)
      File "c:\program files\python34\lib\zipfile.py", line 1084, in getinfo
        'There is no item named %r in the archive' % name)
    KeyError: "There is no item named '_frida.pyd' in the archive"

    ----------------------------------------
Command ""c:\program files\python34\python.exe" -u -c "import setuptools, tokenize;__file__='C:\\Users\\Kermit\\AppData\\Local\\Temp\\pip-build-kopnyw58\\frida\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\Kermit\AppData\Local\Temp\pip-7l3e3k_6-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\Kermit\AppData\Local\Temp\pip-build-kopnyw58\frida\
```

Does anyone has any idea ?

**Comment by oleavr** [comment]
Thanks for reporting. Just pushed [some improvements](https://github.com/frida/frida-python/commit/c188bbf6228a34e3efe18666784b9aedafad1ed0) to frida-python's `setup.py` and tagged 7.1.7. Please retry once the builders have finished chewing on this new release and let me know if this resolves the issue.

**Comment by P3pito** [comment]
...

Man you absolutely rock ! You corrected it 10 minutes after my report.

It worked just fine ! Thank you : )

**Comment by oleavr** [comment]
Thanks for testing – glad to hear that! :) Found another issue and fixed it in 7.1.8, that should hopefully be all. Enjoy! :)
